src/AVFoundation/AVPlayerItemVideoOutput.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+using XamCore.Foundation;
+using XamCore.CoreFoundation;
+using XamCore.ObjCRuntime;
+using XamCore.AudioToolbox;
+
+namespace XamCore.AVFoundation {
+	public partial class AVPlayerItemVideoOutput {
+
+		enum InitMode {
+			PixelAttributes,
+			OutputSettings
+		} 
+
+		AVPlayerItemVideoOutput (NSDictionary data, AVPlayerItemVideoOutput.InitMode mode) : this (IntPtr.Zero)
+		{
+			switch (mode) {
+			case InitMode.PixelAttributes:
+				Handle = _FromPixelBufferAttributes (data);
+				break;
+			case InitMode.OutputSettings:
+				Handle = _FromOutputSettings (data);
+				break;
+			default:
+				throw new ArgumentException (nameof (mode));
+			}
+		}
+
+		[DesignatedInitializer]
+		[Obsolete ("Please use the constructor that uses one of the available StrongDictionaries.")]
```

Comment:
an `[Advice]` (with a message) or `[Advanced]` (without) would be better than `[Obsolete]` - we should not add _really_ obsolete API and the other two attributes allow us to tell developers this is not the best API that can be used.